packages/trace-viewer/src/ui/networkResourceDetails.tsx

```diff
@@ -128,11 +132,33 @@ const BodyTab: React.FunctionComponent<{
 
   return <div className='network-request-details-tab'>
     {!resource.response.content._sha1 && <div>Response body is not available for this request.</div>}
+    {responseBody && responseBody.font && <FontPreview font={responseBody.font} />}
     {responseBody && responseBody.dataUrl && <img draggable='false' src={responseBody.dataUrl} />}
     {responseBody && responseBody.text && <CodeMirrorWrapper text={responseBody.text} mimeType={responseBody.mimeType} readOnly lineNumbers={true}/>}
   </div>;
 };
 
+const FontPreview: React.FunctionComponent<{
+  font: BinaryData;
+}> = ({ font }) => {
+  React.useEffect(() => {
+    const fontFace = new FontFace('font-preview', font);
```

Comment:
```suggestion
    // note: constant font family name will lead to bugs
    // when displaying two font previews.
    const fontFace = new FontFace('font-preview', font);
```

just noting this down for future reference. Don't think we need to change anything though.

Reply:
Absolutely! But I'd still like to note this down, in case we ever decided to display two font faces at a time. The fact that there's a bug in here if the component is used twice isn't very obvious, so the comment hopefully prevents us scratching our heads :D